src/ncp/ncp_base.cpp

```diff
@@ -1091,6 +1092,7 @@ void NcpBase::LinkRawReceiveDone(otRadioFrame *aFrame, otError aError)
     SuccessOrExit(error = OutboundFrameSend());
 
 exit:
+    OT_UNUSED_VARIABLE(error);
```

Comment:
If we want to keep these around for debugging, I think it is better to have them used in a log statement.  Then the variable is useful whether or not using a debugger is possible.  Thoughts?